docs/en/rules/Azure.ACR.ExportPolicy.md

```diff
@@ -0,0 +1,119 @@
+
+## SYNOPSIS
+
+Disable export of artifacts from Azure container registry to ensure data is accessed solely via the data plane.
+
+## DESCRIPTION
+
+Azure Container Registry (ACR) export policy allows copying container images and artifacts to other registries or locations. 
+When the export policy is enabled, data can be moved out of the registry via 'acr import' or 'acr transfer' commands.
+
+To improve security and prevent data exfiltration, the export policy should be disabled.
+This ensures that data in the registry is accessed solely through the data plane using standard Docker commands like 'docker pull'.
+
+Disabling export policy requires that public network access is also disabled.
+This provides additional protection by ensuring the registry is only accessible from private networks.
+
+## RECOMMENDATION
+
+Consider disabling the export policy and public network access for container registries containing sensitive data.
+
+## EXAMPLES
+
+### Configure with Bicep
+
+To deploy registries that pass this rule:
+
+- Set `properties.policies.exportPolicy.status` to `disabled`.
+- Set `properties.publicNetworkAccess` to `Disabled`.
+
+For example:
+
+```bicep
+resource containerRegistry 'Microsoft.ContainerRegistry/registries@2023-07-01' = {
+  name: registryName
+  location: location
+  sku: {
+    name: 'Premium'
+  }
+  properties: {
+    adminUserEnabled: false
+    publicNetworkAccess: 'Disabled'
+    policies: {
+      exportPolicy: {
+        status: 'disabled'
+      }
+    }
+  }
+}
+```
+
+### Configure with Azure template
+
+To deploy registries that pass this rule:
+
+- Set `properties.policies.exportPolicy.status` to `disabled`.
+- Set `properties.publicNetworkAccess` to `Disabled`.
+
+For example:
+
+```json
+{
+  "type": "Microsoft.ContainerRegistry/registries",
+  "apiVersion": "2023-07-01",
+  "name": "[parameters('registryName')]",
+  "location": "[parameters('location')]",
+  "sku": {
+    "name": "Premium"
+  },
+  "properties": {
+    "adminUserEnabled": false,
+    "publicNetworkAccess": "Disabled",
+    "policies": {
+      "exportPolicy": {
+        "status": "disabled"
+      }
+    }
+  }
+}
+```
+
+### Configure with Azure CLI
+
+```bash
+az acr update --name MyRegistry --public-network-enabled false
+az acr config export-policy update --registry MyRegistry --status disabled
+```
+
+### Configure with Azure PowerShell
+
+```powershell
+Update-AzContainerRegistry -Name MyRegistry -ResourceGroupName MyResourceGroup -PublicNetworkAccess Disabled
+```
+
+## NOTES
+
+An Azure container registry with export policy disabled:
+
+- Prevents copying registry data using ACR import/export commands.
+- Requires public network access to be disabled.
+- Allows standard Docker operations (pull, push) through private endpoints.
+- Is available for Premium tier registries only.
+
+## LINKS
+
+- [Data loss prevention for Azure Container Registry](https://learn.microsoft.com/en-gb/azure/container-registry/data-loss-prevention)
```

Comment:
Add an additional link first in the list for: `[SE:08 Hardening resources](https://learn.microsoft.com/en-us/azure/well-architected/security/harden-resources)` which is the reference for the WAF pillar this aligns to.